<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LastRoundBar</class>
 <widget class="QWidget" name="LastRoundBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(197, 195, 196);
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>220</width>
     <height>180</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>216</width>
     <height>176</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
</string>
   </property>
   <widget class="QLabel" name="label_M">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Radio Space Bold</family>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFCC01;
border-left: 1px solid black;
border-top: 1px solid black;</string>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_S">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>60</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Radio Space Bold</family>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFCC01;
border-top: 1px solid black;</string>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Min">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>70</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Radio Space Bold</family>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFCC01;
border-top: 1px solid black;
border-right: 1px solid black;</string>
    </property>
    <property name="text">
     <string>min.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>21</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Radio Space Bold</family>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFCC01;
border-top: 1px solid black;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Team">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>220</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Radio Space Bold</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
border-left: 1px solid black;
border-right: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Rounds">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>220</width>
      <height>53</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Radio Space Bold</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 6);
border-left: 1px solid black;
border-right: 1px solid black;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>136</y>
      <width>220</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Radio Space Bold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFCC01;
border-left: 1px solid black;
border-right: 1px solid black;
border-bottom: 1px solid black;</string>
    </property>
    <property name="text">
     <string>POSLEDNI PRIPSANE KOLO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
